internal/modifier/templates/panic_error.go.tmpl

```diff
@@ -0,0 +1,55 @@
+{{- define "panicError" -}}
+{{- $cff := import "go.uber.org/cff" -}}
+{{ template "stacktrace" }}
+err = {{ $cff }}.PanicError{
+    Value:      recovered,
+    Stacktrace: stacktrace,
+}
+{{- end -}}
+
+{{- define "stacktrace" -}}
+{{- $runtime := import "runtime" -}}
+{{ template "funcFormatFunction" }}
+{{ template "funcGetStack" }}
+pc := make([]uintptr, 64)
+{{- /* skipping 2 in the callers, which are: */ -}}
+{{- /* - runtime.Callers */ -}}
+{{- /* - caller of Callers (self) */ -}}
+{{- /* because the panic should not be in any of these callers */}}
```

Comment:
It is. 
```suggestion
{{- /*
  skipping 2 in the callers, which are:
  - runtime.Callers
  - caller of Callers (self)
  because the panic should not be in any of these callers
*/}}
```